.scannerwork/report-task.txt

```diff
@@ -0,0 +1,6 @@
+projectKey=bebbo
+serverUrl=http://localhost:9000
+serverVersion=10.6.0.92116
+dashboardUrl=http://localhost:9000/dashboard?id=bebbo
+ceTaskId=5101a772-dfc9-4e3c-b3d1-0292b54796c1
+ceTaskUrl=http://localhost:9000/api/ce/task?id=5101a772-dfc9-4e3c-b3d1-0292b54796c1
```

Comment:
Keep these files out of repository, you can add entire folder to .gitignore file:

.scannerwork/